client/my-sites/email/email-providers-stacked-comparison/email-provider-stacked-card/style.scss

```
@@ -249,19 +246,100 @@ h2.action-panel__title {
 	color: var( --color-neutral-70 );
 }
 
-.email-provider-stacked-card__header {
-	display: flex;
-	flex-direction: row;
-	flex-wrap: wrap;
-	justify-content: space-between;
+.email-providers-stacked-comparison__provider-card {
+	.email-provider-stacked-card__header {
+		display: flex;
+		flex-direction: column;
+		flex-wrap: wrap;
+
+		.email-provider-stacked-card__provider-card-main-details {
+			margin-top: 12px;
+			width: calc( 100% - 10px );
+			margin-left: 10px;
+			@include break-large {
```

Nit: it would be nice to have a line break between these two sections.
```suggestion
			margin-left: 10px;

			@include break-large {
```